terraform/stacks/unimelb/data_archive/byob_ica_v2.tf

```diff
@@ -616,6 +616,36 @@ data "aws_iam_policy_document" "development_data" {
       "${aws_s3_bucket.development_data.arn}/*",
     ]
   }
+  statement {
+    sid = "nextflow_batch"
+    principals {
+      type        = "AWS"
+      // arn:aws:iam::843407916570:role/nextflow/oncoanalyser-pipeline-batch-instance-role
+      identifiers = [
+        "arn:aws:iam::${local.account_id_dev}:role/nextflow-oncoanalyser-pipeline-batch-instance-role",
+        "arn:aws:iam::${local.account_id_dev}:role/nextflow-oncoanalyser-task-batch-instance-role",
+        "arn:aws:iam::${local.account_id_dev}:role/nextflow-sash-pipeline-batch-instance-role",
+        "arn:aws:iam::${local.account_id_dev}:role/nextflow-sash-task-batch-instance-role",
+      ]
+    }
+    actions = [
+      "s3:GetBucketLocation",
+      "s3:GetObject",
```

Comment:
No `PutObject`?

Reply:
Ah sorry, copied from the filemanager policy

Reply:
Wrong statement?

Reply:
Sorry, resolved now